src/preferences/configobject.h

```diff
@@ -155,6 +156,12 @@ template <class ValueType> class ConfigObject {
     // values. ResultType is serialized to string on a per-type basis.
     template <class ResultType>
     void setValue(const ConfigKey& key, const ResultType& value);
+    template<class EnumType>
+    requires std::is_enum_v<EnumType>
```

Comment:
>  The alternative would be to merge that to main, because this is not a last minute bug fix.

I'd prefer merging to main then. The error messages generated when using `enable_if` are much uglier than the ones generated by `requires` clauses.

Reply:
OK